### PR TITLE
11470

### PR DESCRIPTION
**Pull request by ecchol**
# 11470

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [Y] All filenames are lower case (including folders, images, files, etc.)
- [Y] Filenames are descriptive
- [Y] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [Y] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [N] Image references in markdown contain an alternate text

### CODE REVIEW

**Comment by hope-fisher** [comment]
Hi team,

All namez in the folder structure need to be in lowercase. Please update **Troubleshooting** and all references to it (as in the manifest). All other changes look fine, so please ping me when this is done so I can review/approve.

**Comment by ecchol** [comment]
> Hi team,
> 
> All namez in the folder structure need to be in lowercase. Please update **Troubleshooting** and all references to it (as in the manifest). All other changes look fine, so please ping me when this is done so I can review/approve.

**Comment by hope-fisher** [comment]
Hi Rahul, I see you've updated the reference to the troubleshooting filepaths, but the folder itself is still uppercase. Can you push your final commit to update that?
Screenshot 2024-01-12 at 10 48 52 AM

File: ecc-developer-hol/query/useeccdashboards/useeccdashboards.md
Comment (ramumg) [inline ecc-developer-hol/query/useeccdashboards/useeccdashboards.md:1]: Please make sure all images are displayed. It looks like ecc.png and val1.png are not being displayed properly.

**Review by ramumg — CHANGES_REQUESTED** [review]

**Comment by ramumg** [comment]
Hi @ecchol, I wanted to follow up with you about this request.

File: ecc-developer-hol/query/Troubleshooting/start-stop-ecc.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+In this lab, you will:
+* Learn how to start and stop ECC
+
+### Prerequisites 
+
+This lab assumes you have:
+* Completed all previous labs successfully 
+
+##
```
Comment (ramumg) [inline ecc-developer-hol/query/Troubleshooting/start-stop-ecc.md]: This shouldn't be here.

**Review by ramumg — DISMISSED** [review]

**Review by ramumg — APPROVED** [review]

**Pull request merged**